src/_derive/mod.rs

```diff
@@ -140,8 +140,10 @@
 //! - e.g. `#[command(arg_required_else_help(true))]` would translate to `cmd.arg_required_else_help(true)`
 //!
 //! **Magic attributes:**
-//! - `name  = <expr>`: [`Command::name`][crate::Command::name]
+//! - `name = <expr>`: [`Command::name`][crate::Command::name]
 //!   - When not present: [package `name`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-name-field) (if on [`Parser`][crate::Parser] container), variant name (if on [`Subcommand`][crate::Subcommand] variant)
+//! - `value_name = <expr>`: [`Arg::value_name`][crate::Arg::value_name]
+//!   - When not present: [package `name`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-name-field)
```

Comment:
`value_name` is not a `command` attribute

Reply:
```
#[derive(Parser, Debug)]
#[command(version)]
#[command(about = "Diff files using patience algorithm")]
struct Args {
  /// Path to old file or `-` for stdin.
  #[clap(name = "OLD_FILE")]
  lhs: PathBuf,

  /// Path to new file or `-` for stdin.
  #[clap(name = "NEW_FILE")]
  rhs: PathBuf,

  /// Display NUM lines of unchanged context before and after changes
  #[clap(short, long, value_name = "NUM", default_value_t = 3)]
  context: usize,
}
```

does what I want in terms of modifying the help text.  Where should I document that?

Reply:
https://docs.rs/clap/latest/clap/_derive/index.html#arggroup-attributes ?

Reply:
I guess this qualifiers as a "raw attribute", but it really took me a while to find it, so I would like to do something to make it easier to find for new users.

Reply:
That problem is being discussed in https://github.com/clap-rs/clap/discussions/4090

Reply:
as a quick workaround for this point, I suggest adding a usage of `value_name` in the overview https://docs.rs/clap/latest/clap/_derive/index.html#overview

Would that patch be of interest?

Reply:
Each person will have their own pet attribute they would want highlighted on an example like that.  The case would need to be made for why that attribute over any other.  Most likely, that change won't happen.